docs/projects/jupiter_sdr/index.rst

```diff
@@ -0,0 +1,308 @@
+.. _jupiter_sdr:
+
+JUPITER SDR HDL Project
+===============================================================================
+
+Overview
+-------------------------------------------------------------------------------
+
+This design allows controlling, receiving and transmitting sample stream
+from/to an ADRV9001/ADRV9002 device through two independent source-synchronous
```

Comment:
```suggestion
from/ to an :adi:`ADRV9001`/ :adi:`ADRV9002` device through two independent source-synchronous
```

Reply:
ADRV9001 is not a device, is the family, to which the ADRV9002 belongs.
Regarding the lines we are talking. The "ADRV9001/ADRV9002" was meant to remove confusion, that axi_adrv9001 is used with ADRV9002. Not the best of ideas. I'm open for suggestions.
But I wouldn't add the ADRV9001 as a link, which does not exist.

Reply:
Right, got it.